task-requests/details/script.js

```
@@ -529,6 +539,9 @@ const renderTaskRequest = async () => {
     taskRequest = await fetchTaskRequest();
     isSuperUser = await getIsSuperUser();
     taskRequestSkeleton.classList.add('hidden');
+    if (!taskRequest) {
+      throw new Error('Task not found');
```

We are already doing error handling in fetchTaskRequest and showErrorMessage function. Do we need to throw this error here?

---

Yes, the throw is helpful to prevent continuing with an invalid state.

---

Replace the 'Task not found' with the constant you introduced in constants.js